pkg/cvo/availableupdates.go

```diff
@@ -367,3 +367,7 @@ func evaluateConditionalUpdate(ctx context.Context, conditionalUpdate *configv1.
 	recommended.Message = strings.Join(messages, "\n\n")
 	return recommended
 }
+
+func (optr *Operator) conditionalUpdateEvaluated(ctx context.Context, condition []configv1.ClusterCondition, match bool, err error) {
+	FIXME: lock the structure, find any updates with risks matching this set, and... re-evaluate them?  Hmm...
```

Comment:
We don't currently have per-risk `Match` conditions, although we could grow those.  That makes it hard to go from "I just figured out that risk A matched!" (or "...did not match") to "so target 4.y.z is not recommended", because you also need to remember whether 4.y.z's riskB, etc. matched.

Reply:
I'm dropping the callback approach in favor of requeueing, see 95c85b72.